x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/logs/logs_tab_content.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { Filter } from '@kbn/es-query';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { SnapshotNode } from '../../../../../../../common/http_api';
+import { LogStream } from '../../../../../../components/log_stream';
+import { useHostsViewContext } from '../../../hooks/use_hosts_view';
+import { useUnifiedSearchContext } from '../../../hooks/use_unified_search';
+import { useLogsSearchUrlState } from '../../../hooks/use_logs_search_url_state';
+import { LogsLinkToStream } from './logs_link_to_stream';
+import { LogsSearchBar } from './logs_search_bar';
+
+export const LogsTabContent = () => {
+  const [filterQuery] = useLogsSearchUrlState();
+  const { getDateRangeAsTimestamp } = useUnifiedSearchContext();
+  const { from, to } = useMemo(() => getDateRangeAsTimestamp(), [getDateRangeAsTimestamp]);
+  const { hostNodes } = useHostsViewContext();
+
+  const hostsFilterQuery = useMemo(() => createHostsFilter(hostNodes), [hostNodes]);
+
+  const logsLinkToStreamQuery = useMemo(() => {
+    const hostsFilterQueryParam = createHostsFilterQueryParam(hostNodes);
+    return `${filterQuery.query ? filterQuery.query + ' and ' : ''}${hostsFilterQueryParam}`;
+  }, [filterQuery.query, hostNodes]);
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="m" data-test-subj="hostsView-logs">
+      <EuiFlexGroup gutterSize={'m'} alignItems={'center'} responsive={false}>
+        <EuiFlexItem>
+          <LogsSearchBar />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <LogsLinkToStream startTimestamp={from} endTimestamp={to} query={logsLinkToStreamQuery} />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      {hostNodes.length ? (
```

Comment:
> Should we have a loading indicator here? When the hosts are still loading, the Logs section just shows a blank space

@crespocarlos I guess it does show a loading indicator 
<img width="1476" alt="Screenshot 2023-03-21 at 21 36 06" src="https://user-images.githubusercontent.com/11225826/226746797-e699d965-e7bb-4711-be20-8640234599e9.png">

Reply:
@mohamedhamed-ahmed , I don't know what environment your local kibana is pointed to, but I'm testing it on an oblt-cli cluster, and on the video below (from 0:14 to 0:16) you can see that while the hosts are loading, the Logs section remains empty. The loading indicator is only shown after the hosts have been loaded. Depending on how slow it is to load the hosts, this could be more noticeable.

https://user-images.githubusercontent.com/2767137/226913604-1422c53f-1361-4db5-95a8-5ce86cfcc6bc.mov

Reply:
I see what you mean, fixed it and pushed the new changes 👍